db/migrate/20140114144500_signin_permissions_must_be_lowercase.rb

```diff
@@ -1,5 +1,5 @@
 class SigninPermissionsMustBeLowercase < ActiveRecord::Migration
   def up
-    SupportedPermission.update_all({ name: 'signin' }, { name: 'Signin' })
+    SupportedPermission.where({ name: 'Signin' }).update_all({ name: 'signin' })
```

Comment:
I know you're doing a like-for-like here, but could we drop the surrounding braces for the option-style arguments? They're not necessary.